specification/advisor/resource-manager/Microsoft.Advisor/stable/2020-01-01/advisor.json

```diff
@@ -142,11 +144,26 @@
           },
           {
             "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "name": "configurationName",
+            "in": "path",
+            "description": "Advisor configuration name. Value must be 'default'",
+            "required": true,
+            "type": "string",
+            "enum": ["default"],
+            "x-ms-enum": {
+              "name": "ConfigurationName",
+              "modelAsString": true
+            }
           }
         ],
         "responses": {
```

Comment:
For all API responses, please add a "default" response to capture error cases. It should have a schema consistent with the [common error contract](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/common-api-details.md#error-response-content). 
Here is an example:
https://github.com/Azure/azure-rest-api-specs/blob/0cf70260108cb40bc68d408674c187ed065f8c64/specification/batch/resource-manager/Microsoft.Batch/stable/2019-08-01/BatchManagement.json#L105

If it's too costly to update all APIs, let's at least address changed apis in this PR.

Reply:
I want to fix this for config API only. Is it possible to add a new api-version for config API, so that we don't have to add a new api-version whenever we make this fix to other APIs? If it is, will SDKs call other APIs with old api-version and call config API with the new version?